agent/js/package.json

```diff
@@ -29,11 +39,28 @@
   },
   "exports": {
     "require": "./dist/index.js",
```

Comment:
I'm not sure if the `require` here is necessary - the exports from the package should be handled by the top level fields

```json
"main": "dist/cjs/index.js",
"module": "dist/esm/index.js",
"umd": "dist/umd/index.js",
```

if the build process correctly outputs them.

Reply:
The build process does not output them. I had issues with the tsup build process before which is why I had the other process, but I haven't experimented with this one to create those outputs.

Reply:
Do we even need the `exports` section?

Reply:
:+1: